.github/workflows/test.yml

```diff
@@ -27,6 +27,20 @@ jobs:
     - uses: actions/setup-node@v4
       with:
         node-version: lts/*
+    - run: |
+        wget https://apt.llvm.org/llvm.sh
+        chmod +x llvm.sh
+        sudo ./llvm.sh 20
+      if: ${{ matrix.platform == 'linux' }}
+    - run: choco upgrade llvm
+      if: ${{ matrix.platform == 'win32' }}
+    - run: |
+        brew install llvm
+        echo "/opt/homebrew/opt/llvm/bin" >> "$GITHUB_PATH"
+        echo "LDFLAGS=-L/opt/homebrew/opt/llvm/lib" >> "$GITHUB_ENV"
+        echo "CPPFLAGS=-I/opt/homebrew/opt/llvm/include" >> "$GITHUB_ENV"
```

Comment:
```suggestion
```

With https://github.com/holepunchto/cmake-toolchains/pull/1 `bare-make` will look for and use the Homebrew version of LLVM when available.